deployment_script.sh

```diff
@@ -2,7 +2,7 @@
 
 rm -f -r ./package
 rm -f deployment-package.zip
-pip3.9 install --target ./package -r requirements.txt
+pip3.12 install --no-dependencies --target ./package -r requirements.txt
```

Comment:
Ah this is probably why requests, etc. wasn't being installed -- remove this now that nypl-py-utils v1.3.1 is in prod